### PR TITLE
Bug fix for crash in QScriptEngine::reportAdditionalMemoryCost()

### PR DESCRIPTION
**Pull request by hyperlogic**
This fix requires a patch to qt.  This build includs a new pre-built Qt 5.12.3 with the patch applied.
See tools/qt5-builder/qtscript-crash-fix.patch to see actual patch applied.

* Touched bullet3 portfile to create new vckpg hash
* New urls for qt assets which include qscript patch hifi_vcpkg.py
* Updated tools/qt5-builder/ with qscript patch and README.md directions.

https://highfidelity.atlassian.net/browse/BUGZ-46

A qt bug was filed to hopefully get this bug fixed upstream.
https://bugreports.qt.io/browse/QTBUG-76176

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19743/HighFidelity-Beta-PR15669-154fd39.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19743/HighFidelity-Quest-Beta-PR15669-154fd39.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19743/HighFidelity-Beta-PR15669-154fd39.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19743/HighFidelity-Beta-Interface-PR15669-154fd39.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19743/HighFidelity-Beta-PR15669-154fd39.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19743/HighFidelity-Beta-Interface-PR15669-154fd39.exe

File: hifi_vcpkg.py [hunk 1]
```diff
@@ -254,9 +254,9 @@ def installQt(self):
 
             url = 'NOT DEFINED'
             if platform.system() == 'Windows':
-                url = 'https://hifi-public.s3.amazonaws.com/dependencies/vcpkg/qt5-install-5.12.3-windows2.tar.gz'
+                url = 'https://hifi-public.s3.amazonaws.com/dependencies/vcpkg/qt5-install-5.12.3-windows3.tar.gz'
             elif platform.system() == 'Darwin':
-                url = 'https://hifi-public.s3.amazonaws.com/dependencies/vcpkg/qt5-install-5.12.3-macos2.tar.gz'
+                url = 'https://hifi-public.s3.amazonaws.com/dependencies/vcpkg/qt5-install-5.12.3-macos3.tar.gz'
             elif platform.system() == 'Linux':
```
Comment (AndrewMeadows) [inline hifi_vcpkg.py]: I don't see a new binary for **linux** so this crash won't be fixed there?  I should patch and build my own Qt libs for that platform?

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19749/HighFidelity-Beta-PR15669-e1534b3.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19749/HighFidelity-Beta-Interface-PR15669-e1534b3.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19749/HighFidelity-Beta-PR15669-e1534b3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19749/HighFidelity-Beta-Interface-PR15669-e1534b3.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19749/HighFidelity-Beta-PR15669-e1534b3.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19749/HighFidelity-Quest-Beta-PR15669-e1534b3.apk)

**Pull request merged**